### PR TITLE
Bond list

### PR DESCRIPTION
**Pull request by mariuszk98**

### CODE REVIEW

File: src/components/Bonds/BondList/BondList.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { withKnobs } from '@storybook/addon-knobs'
+import { storiesOf } from '@storybook/react'
+import React from 'react'
+import BondList from './BondList'
+
+storiesOf('bonds/BondList', module)
+  .addDecorator(withKnobs)
+  .add('BondList', () => {
+    const [data] = React.useState(
```
Comment (awojciak) [inline src/components/Bonds/BondList/BondList.stories.tsx:9]: useState is unnecessary here, data is not changing

File: src/components/Bonds/BondItem/BondItem.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            </Typography>
+          </Grid>
+        </Grid>
+      </Grid>
+      <Grid className={classes.price}>
+        {printBN(new BN(price), decimals)} {symbol} per {secondSymbol}
+      </Grid>
+      <Grid className={classes.roi}>+{roiPercent}%</Grid>
+      {!isExSmall ? (
+        <>
```
Comment (awojciak) [inline src/components/Bonds/BondItem/BondItem.tsx:63]: fragments are not necessary if there is only one node on top level

File: src/components/Bonds/BondItem/BondItem.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+          <Typography className={classes.purchased}>${purchased}</Typography>
+        </>
+      ) : (
+        <></>
```
Comment (awojciak) [inline src/components/Bonds/BondItem/BondItem.tsx:67]: just use null

**Comment by PawelKrawczyk96** [comment]
image
maybe move pair column to the left below 600px , ROI don't have space a little

File: src/components/Bonds/BondItem/BondItem.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { printBN } from '@consts/utils'
+import { Button, Grid, Typography, useMediaQuery } from '@material-ui/core'
+import { BN } from '@project-serum/anchor'
+import { colors, theme } from '@static/theme'
+import { useStyles } from './style'
+
+interface BondItem {
```
Comment (PawelKrawczyk96) [inline src/components/Bonds/BondItem/BondItem.tsx:8]: change name to IBondItem (interfaces have 'I' on the beginning)

**Review by PawelKrawczyk96 — CHANGES_REQUESTED** [review]

**Review by awojciak — CHANGES_REQUESTED** [review]
1) fix remaining lint issues
2) storybook: 
obraz
mockup:
obraz
3) 
obraz
there is something wrong with font

**Review by awojciak — CHANGES_REQUESTED** [review]
1) there is still no gradient in button
2) https://www.figma.com/file/LsofkIRxZQIgGo1p18raOy/App?node-id=2905%3A7815 - hover on button doesn't look like that in storybook, fix this
3) 
obraz
i think header and content in this column should be centered

**Review by awojciak — APPROVED** [review]

**Pull request merged**